studio/src/mas-fragment-render.js

```diff
@@ -1,61 +1,148 @@
+            // Set the drag effect
+            event.dataTransfer.effectAllowed = 'copy';
+
+            // Add a class to indicate dragging
+            event.currentTarget
+                .closest('.render-fragment')
+                .classList.add('dragging');
+        } catch (error) {
+            console.error('Error setting drag data:', error);
+            event.preventDefault();
+        }
+    }
+
+    handleDragEnd(event) {
+        // Remove the dragging class
+        event.currentTarget
+            .closest('.render-fragment')
+            .classList.remove('dragging');
     }
 
     get selectionOverlay() {
-        if (!this.selecting.value) return nothing;
+        if (!Store.selecting.value) return nothing;
         return html`<div class="overlay" @click="${this.select}">
             ${this.selected
                 ? html`<sp-icon-remove slot="icon"></sp-icon-remove>`
                 : html`<sp-icon-add slot="icon"></sp-icon-add>`}
         </div>`;
     }
 
+    get merchCard() {
+        return html`<merch-card slot="trigger">
+            <aem-fragment
+                fragment="${this.fragment.id}"
+                ims
+                author
+            ></aem-fragment>
+            ${this.selectionOverlay}
+        </merch-card>`;
+    }
+
+    get unknown() {
+        const label = this.fragment.fields.find(
+            (field) => field.name === 'label',
+        )?.values[0];
+        return html`<div class="unknown-fragment" slot="trigger">
+            <sp-icon-document-fragment></sp-icon-document-fragment> ${label}
+            ${this.selectionOverlay}
+            <p class="model-name">${this.fragment.title}</p>
+        </div>`;
+    }
+
     render() {
-        return html`<div class="render-card">
-            <div class="render-card-header">
-                <div class="render-card-actions"></div>
+        return html`<div class="render-fragment">
+            <div class="render-fragment-header">
+                <div class="render-fragment-actions"></div>
                 <mas-fragment-status
-                    variant=${this.fragment.value.statusVariant}
+                    variant=${this.fragment.statusVariant}
                 ></mas-fragment-status>
             </div>
-            <overlay-trigger placement="top">
-                <merch-card slot="trigger">
-                    <aem-fragment
-                        fragment="${this.fragment.value.id}"
-                        ims
-                        author
-                    ></aem-fragment>
-                    ${this.selectionOverlay} </merch-card
-                ><sp-tooltip slot="hover-content" placement="top"
-                    >Double click the card to start editing.</sp-tooltip
-                >
-            </overlay-trigger>
+            <div
+                class="render-fragment-content"
+                draggable="true"
+                @dragstart=${this.handleDragStart}
+                @dragend=${this.handleDragEnd}
```

Comment:
We should probably add accessibility support
```suggestion
                @dragend=${this.handleDragEnd}
                    aria-grabbed="${this.isDragging}"
    aria-label="Draggable fragment ${this.fragment?.title || ''}"
```